src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/UserInviteSubmission.cs

```diff
@@ -0,0 +1,10 @@
+namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels
+{
+    public class UserInviteSubmission
```

Comment:
I think everywhere else we are using Request for the naming so this would be `UserInviteRequest`